modules/gapi/include/opencv2/gapi/gtransform.hpp

```diff
@@ -16,13 +16,16 @@
 #include <opencv2/gapi/garg.hpp>
 #include <opencv2/gapi/gtype_traits.hpp>
 #include <opencv2/gapi/util/compiler_hints.hpp>
+#include <opencv2/gapi/gcomputation.hpp>
 
 namespace cv
 {
 
 struct GAPI_EXPORTS GTransform
 {
-    using F = std::function<GArgs(const GArgs &)>;
+    // FIXME: consider another simplified
+    // class instead of GComputation
+    using F = std::function<GComputation()>;
 
     std::string description;
     F pattern;
```

Comment:
What is the reason to have this as a callback? Maybe store a cv::GComputation as-is instead?

Reply:
Will close after Alexey returns from a leave.

Reply:
Since this function doesn't operate with GArgs anymore there is no reason to have it as a callback. Can be easily changed if needed, just let me know.